onnxruntime/core/optimizer/div_mul_fusion.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer.h"
+
+namespace onnxruntime {
+/**
+@Class DivMulFusion
+
+Transform that fuses two Div -> Mul nodes to a single Div node
+when the first input to Div is 1.
+1 / x1 *  x2 -> x2 / x1
+
+*/
+class DivMulFusion : public GraphTransformer {
```

Comment:
This fusion can be written as a RewriteRule, since the change is very localized.

See other RewriteRule for more examples, e.g. ConvAddFusion, BatchNormAddFusion...